lib/core/src/server/loadCustomBabelConfig.js

```diff
@@ -25,6 +26,17 @@ function loadFromPath(babelConfigPath) {
       logger.error(`=> Error parsing .babelrc file: ${e.message}`);
       throw e;
     }
+  } else {
+    try {
+      // eslint-disable-next-line import/no-dynamic-require,global-require
+      const jsConfig = require(path.resolve(babelConfigDir, 'babel.config.js'));
```

Comment:
I think it's better to use `serverRequire` to support also `babel.config.ts`

Reply:
Sure, I think I'll just close this one in favor of yours

Reply:
Cool =) Would love to hear your thoughts there